examples/rapids/mlflow_project/notebooks/rapids_mlflow.ipynb

```diff
@@ -0,0 +1,301 @@
+{
+ "cells": [
```

Comment:
Can we run this notebook on Google Colab?

Reply:
You can run in colab, and install rapids using the csp helpers (https://github.com/rapidsai/rapidsai-csp-utils); however, it does require at least a Pascal level GPU, which may not be satisfied if colab gives you a k80.